Source/DataSources/BoundingSphereState.js

```diff
@@ -0,0 +1,34 @@
+/*global define*/
+define([
+        '../Core/freezeObject'
+    ], function(
+        freezeObject) {
+    "use strict";
+
+    /**
+     * The state of a BoundingSphere computation being performed by a {@link Visualizer}.
+     * @exports BoundingSphereState
+     */
+    var BoundingSphereState = {
```

Comment:
I had something generic at first, but I ended up switching to this because I had trouble coming up with a good name and I'm not sure how reusable it would be anyway.  This could end up private if we really wanted it to (see below).